setup.py

```diff
@@ -0,0 +1,16 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+from setuptools import find_packages, setup
+
+setup(
+    name="omdata",
+    version="0.1.0",
+    description="Code for generating OMOL input configurations",
+    url="http://github.com/Open-Catalyst-Project/om-data",
+    packages=find_packages(),
```

Comment:
Perhaps worthwhile to try being a bit pedantic about versioning early on just as a matter of precaution. We can put `quacc[sella]>=0.7.1` in the list of packages and then make a separate `requirements.txt` file listing the explicit versions used in the production efforts that way it is truly reproducible internally and externally. In that file, we should include specific versions of all primary imports, e.g. `sella`, `quacc`, `ase` (in this case, the commit used), etc.